Makefile

```diff
@@ -40,8 +43,10 @@ tests: .install-poetry
 build: .install-poetry
 	poetry build
 
-release:
-	poetry version ${version}
+release: .install-poetry
+	@echo "Please remember to update the CHANGELOG.md, before creating the release tag in GitHub"
```

Comment:
I think a more correct wording would be '... before tagging the release'. You don't need to use Github for tagging a commit (at least, I never do that. I just run `git tag <version>`.

Reply:
done